auditbeat/module/file_integrity/event_linux.go

```diff
@@ -41,7 +42,9 @@ func NewEventFromEbpfEvent(
 		path, target string
 		action       Action
 		metadata     Metadata
+		process      Process
 		err          error
+		errors       = make([]error, 0)
```

Comment:
Do we need to allocate a zero-length slice? Can we leave it as nil? https://go.dev/wiki/CodeReviewComments#declaring-empty-slices